samples/outlook/10-roaming-settings/roaming-settings.yaml

```diff
@@ -0,0 +1,84 @@
+id: outlook-roaming-settings-roaming-settings
+name: Using add-in settings
+description: 'Gets, sets and saves add-in roaming settings'
+host: OUTLOOK
+api_set: {}
+script:
+    content: |
+        $("#get").click(function() {
```

Comment:
Not sure I understand this one.. map what function to a reference doc? This would be my own function I'd define, so does it matter? I want to keep the samples simpler and easier to read..

Reply:
Thanks for the offline explanation. I got it. As discussed, will make this as a separate commit in next PR.